### DISCUSSION TITLE
login error

### DISCUSSION THREAD

**Discussion by shenyingkui** [rabbitmq/rabbitmq-server#2934] [Other] [answered]
rabbitmq ReferenceError: disable_stats is not defined

installed rabbitmq vesrion 3.8.14.
when login web console ,it print ReferenceError: disable_stats is not defined
![image](https://user-images.githubusercontent.com/31129868/112631164-804e5d00-8e71-11eb-9c30-c14f64c12604.png)

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment, marked as answer]
See under [Management plugin Upgrades](https://www.rabbitmq.com/upgrade.html#unsupported-inplace-upgrade). Clearing your browser cache and reloading the page will help.
